### PR TITLE
Added Taksh's Details and Image

### PR DESCRIPTION
**Pull request by takshch**

### CODE REVIEW

File: ids/mapping.json [hunk 1]
```diff
@@ -13,5 +13,6 @@
   "shihab": "shihabus",
   "swaraj": "swarajpure",
   "sweta": "karnsweta92",
-  "tejas": "tejasberi"
+  "tejas": "tejasberi",
+  "taksh": "takshch"
```
Comment (swarajpure) [inline ids/mapping.json:17]: The ids are supposed to be in alphabetical order, please shift your id line before tejas' id line

File: members/taksh/data.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "takshch",
```
Comment (swarajpure) [inline members/taksh/data.json:2]: This id is supposed to be your RDS id - **taksh** as you've mentioned in the mapping.json file.

File: members/taksh/data.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "first_name": "Taksh",
+  "last_name": "Chanana",
+  "yoe": "College 2nd year student",
```
Comment (swarajpure) [inline members/taksh/data.json:5]: yoe can be 0
company name can be your college name
designation can be **student**

File: ids/mapping.json (hunk 1 above)
Comment (takshch) [reply ids/mapping.json]: Changes are made. Please pr review again.

File: members/taksh/data.json (hunk 2 above)
Comment (takshch) [reply members/taksh/data.json]: Changed to "taksh"

File: members/taksh/data.json (hunk 3 above)
Comment (takshch) [reply members/taksh/data.json]: I don't want to be a student here!.

**Comment by ankushdharkar** [comment]
Thank you for reviewing @swarajpure :)

**Pull request merged**